### PR TITLE
cpuid.md: add page

### PR DESCRIPTION
**Pull request by jsonbruce**
----

- [x] The page (if new), does not already exist in the repo.

- [x] The page (if new), has been added to the correct platform folder:  
      `common/` if it's common to all platforms, `linux/` if it's Linux-specific, and so on.

- [x] The page has 8 or fewer examples.

- [x] The PR is appropriately titled:  
      `<command name>: add page` for new pages, or `<command name>: <description of changes>` for pages being edited

- [x] The page follows the [contributing](https://github.com/tldr-pages/tldr/blob/master/CONTRIBUTING.md) guidelines

### CODE REVIEW

File: pages/linux/cpuid.md [hunk 1]
```diff
@@ -0,0 +1,15 @@
+# cpuid
+
+> Display detailed information about all CPUs.
+
+- Display information for all CPUs:
+
+`cpuid`
+
+- Display information only for the first CPU:
```
Comment (sbrl) [inline pages/linux/cpuid.md]: Is there an option to display info only for, say, the third CPU too?

Comment (jsonbruce) [reply pages/linux/cpuid.md]: No, I haven't found.
I've updated the confusing description.
`Display information only for the current CPU`
Not `the first CPU`.

Comment (sbrl) [reply pages/linux/cpuid.md]: What's meant by `current` cpu?

Comment (jsonbruce) [reply pages/linux/cpuid.md]: `cpuid -1` captures the current cpu on the multiprocessor system,

**Pull request merged**